k8s/base/gateway-demo.yaml

```diff
@@ -3,6 +3,7 @@ kind: SpringCloudGateway
 metadata:
   name: gateway-demo
 spec:
+  ssoEnabled: true
```

Comment:
Now that we have SSO, can you also add the other routes specified in the old `gateway-config.json` from both frontend and backend?